source/docs/power-and-electronics/control-system.rst

```diff
@@ -3,47 +3,52 @@
 Control Systems
 ===============
 
-The FTC control system is based on using Android phones as a ":term:`Robot Controller`" and a ":term:`Driver Station`." The ":term:`Robot Controller`" phone is mounted on the robot and connected by a USB cable to a special "Hub", which in turn is connected to motors, servos, and sensors.
+The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and are connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```

Comment:
```suggestion
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and is connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```

Reply:
This paragraph doesn't really account for the fact that the Robot Controller can itself be a REV Hub. Additionally, the Control Hub does not use Wi-Fi Direct in any way. Instead, it broadcasts a standard Wi-Fi access point.

Reply:
Thoughts:
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and handles motors, servos, sensors, and input from the drivers' controllers. The :term:`Driver Station` remain with the drive team and connects to the :term:`Robot Controller` over WiFi.

Adds some ambiguity to the Robot Controller definition, just giving a very basic overview of its purpose. This leaves the second paragraph for actually describing the hubs. Also fixes up the WiFi Direct mistake.